main/src/ca/uwaterloo/flix/language/ast/Scheme.scala

```diff
@@ -128,21 +127,21 @@ object Scheme {
     * Returns `true` if the given schemes are equivalent.
     */
   // TODO can optimize?
-  def equal(sc1: Scheme, sc2: Scheme, instances: MultiMap[Symbol.ClassSym, ResolvedAst.Instance])(implicit flix: Flix): Boolean = {
-    lessThanEqual(sc1, sc2, instances) && lessThanEqual(sc2, sc1, instances)
+  def equal(sc1: Scheme, sc2: Scheme, classEnv: MultiMap[Symbol.ClassSym, Ast.Instance])(implicit flix: Flix): Boolean = {
```

Comment:
I think I would just really like 

```
MultiMap[Symbol.ClassSym, Ast.Instance]
``` 

to be 

```
Map[Symbol.ClassSym, List[Ast.Instance]]
```

unless there is something fundamental that I have not understood.

Reply:
I used MultiMap for instances in each AST. Should those change as well, or just the classEnvs?

Reply:
Yes, I think those should be changed to. We can do that later if you prefer.

I think that instances come from the AST so they really should be a list.

Reply:
Done!